app/lib/hooks/useOffsetParams.ts

```diff
@@ -52,8 +52,8 @@ export const useOffsetParams = (): OffsetParams => {
     const data: OffsetParams = {};
     inputParams.forEach((param) => {
       if (param === "projectTokens") {
-        const arr = params.getAll("projectTokens");
-        data.projectTokens = arr.length ? arr : undefined;
+        const projectTokens = params.get("projectTokens");
+        data.projectTokens = projectTokens ? projectTokens : undefined;
```

Comment:
Kept the attribute name `projectTokens` to maintain backwards compatibility (instead of being more semantic as `projectToken`.
What are your thoughts on this @Atmosfearful?